sdk/spring-sdk/src/main/java/kalix/springsdk/annotations/JWT.java

```diff
@@ -24,27 +24,28 @@
 public @interface JWT {
 
   enum JwtMethodMode {
-    // No validation.
+    /**
+     * No validation.
+     */
     UNSPECIFIED,
-    // Validate the bearer token.
+
+    /**
+     * Validates that the bearer token is present on the request, in the 'Authorization' header.
+     */
     BEARER_TOKEN,
-    // Validate/sign a token field in the message against the message fields.
```

Comment:
as per https://github.com/lightbend/kalix/issues/7369 we agreed not to deal with messages. Which by the way is the authorization side of JWT. So far we only implemented the authentication side of JWT in the Spring SDK.